### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: docs/developing_running.md [hunk 1]
```diff
@@ -145,3 +143,74 @@ the "follow" mode).
...
+                            serving, all others are included
+* `WORKER_INCLUDE_QUEUES` - a comma separated list regexps describing name
+                            of queues that should be *included* from worker
+                            serving, all others are excluded
+
+This can be especially useful when performing [prioritization or throttling of some particular tasks](http://selinon.readthedocs.io/en/latest/optimization.html#prioritization-of-tasks-and-flows).
+
+## Local deployment
+
+Even though the whole fabric8-analytics is run in Amazon AWS, there basically
```
Comment (tisnik) [inline docs/developing_running.md]: 'there are not any strict limitations' or something similar?

File: docs/developing_running.md [hunk 2]
```diff
@@ -145,3 +143,74 @@ the "follow" mode).
...
+repository.
+
+### Using Minio S3 instead of Amazon S3
+
+To substitute Amazon S3 in local deployment and development setup, there
+was chosen [Minio S3](https://github.com/minio/minio) as an alternative. However
+Minio S3 is not fully compatible alternative (e.g. it does not support versioning
+nor encryption), but it does not restricts basic functionality whatsoever.
+
+If you would like to keep results in PostgreSQL
```
Comment (tisnik) [inline docs/developing_running.md]: something seems to be missing here + it should IMHO go to the next section

**Comment by fridex** [comment]
Thanks @tisnik for the review. The PR was in WIP, now it should be complete. If you have any suggestions on what to add I would appreciate it!

**Comment by krishnapaparaju** [comment]
This looks good. thanks @fridex

**Pull request merged**